src/main/java/nl/tudelft/jpacman/npc/ghost/DispersionBlinky.java

```diff
@@ -0,0 +1,18 @@
+package nl.tudelft.jpacman.npc.ghost;
+
+import nl.tudelft.jpacman.board.Direction;
+
+public class DispersionBlinky extends DispersionMode{
+
+	Ghost g;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://www.codacy.com/app/mathieu-craeye/jpacman-framework_2/file/2532229381/issues/source?bid=3267846&fileBranchId=3284210#l7)